src/components/totals/FundsRaised/index.js

```diff
@@ -58,26 +61,39 @@ module.exports = React.createClass({
     return campaignUids;
   },
 
-  loadCampaigns: function() {
+  loadTotals: function() {
     this.setState({ isLoading: true });
-    campaigns.findByUids(this.setUids(), this.onSuccess);
+
+    if (this.props.pageId && (this.props.campaignUid || this.props.campaignUids.length>0)) {
```

Comment:
Space between operators `this.props.campaignUids.length > 0)`
Check out our JS style guid here https://github.com/everydayhero/styleguide/blob/master/Javascript.md